actors/builtin/miner/miner_test.go

```diff
@@ -4159,6 +4238,26 @@ func (h *actorHarness) withdrawFunds(rt *mock.Runtime, amountRequested, amountWi
 	rt.Verify()
 }
 
+func (h *actorHarness) repayDebt(rt *mock.Runtime, value, expectedRepayedFromVest, expectedRepaidFromBalance abi.TokenAmount) {
```

Comment:
Love the explicit split.  Side note -- one of the next ways to improve miner test harness is to come up with patterns that make this easy to do everywhere.  For example it is very difficult to specify this with the current cron config without heroic contortions in the calling test code.